bip-package-relay.mediawiki

```diff
@@ -0,0 +1,228 @@
+pre>
+  BIP: ?
+  Layer: Peer Services
+  Title: Generic Package Relay
+  Author: Gloria Zhao <gloriajzhao@gmail.com>
+  Comments-Summary:
+  Comments-URI:
+  Status: Draft
+  Type: Standards Track
+  Created: 2022-04-14
+  License: PD
```

Comment:
Before taking this out of draft, please license it under one of the [acceptable licenses](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#Recommended_licenses)

Reply:
Thanks will do! I'm still working on a few changes based on feedback, and then will take this out of draft.